beyond-all-reason/Beyond-All-Reason
# updating Keybindings of Keys (top right button ingame) to gridkeys

When the game is set to use Gridkeys, the Keybindings page of Keys (button top right while ingame) shows an odd mix of Grid and Legacy keys. This update makes all the listed keys to be based on Gridkeys, although the odd mix problem persists if someone is using Legacy or off-grid custom keys.

As Gridkeys is now the default, this is still better, even if the "odd mix" problem persists for some users. Ideally, the whole page would update based on the actual keybinds of the user, but setting that up seems way beyond me.

### Work done
- Changed "Selecting build orders" to Gridkeys.
- Added "select visible+similar" (Q with gridkeys), "command_skip_current" (N) and "command_cancel_last" (ctrl-N).
- Removed sharedialog and move.
- Changed "variable names" in gui_keybind_info.lua and in each interface.json file. Also did some cleanup.
- Capitalized descriptions of unit orders in the english interface.json while I am at it.
- Many minor fixes.
- Updated disclaimer.

#### Addresses Issue(s)
- https://github.com/beyond-all-reason/Beyond-All-Reason/issues/3508


#### Test steps
Start up the game and check out the Keybindings page of Keys (button top right).

To check out the "odd mix" problem, check the Keybindings page before and after switching between Grid and Legacy keys. When on Legacy keys -with this update running - note that the 3rd column has a bunch of wrong infos.

### Screenshots:
#### BEFORE:
![keybindingsweirdmix](https://github.com/user-attachments/assets/0bb62a40-a30d-4756-af0b-26b320e396df)


#### AFTER:
![image](https://github.com/user-attachments/assets/01b355d8-52c9-4a7a-97ce-aedf9b09f71f)
